docs/GettingStartedDocs/IntelSDKPortingGuideLinux.md

```diff
@@ -0,0 +1,311 @@
+# Intel SGX SDK for Linux to Open Enclave SDK Porting Guide
+
+Intel SGX SDK for Linux and Open Enclave SDK share many design principles but
```

Comment:
```suggestion
The [Intel SGX SDK for Linux](https://01.org/intel-software-guard-extensions) and the Open Enclave SDK share many design principles but
```
Besides adding a link, note the grammar fix to add "the".  (By comparison the linked page always uses "the", as do most other OE SDK docs, including lines 282-284 of this PR).